.github/workflows/lint.yml

```diff
@@ -0,0 +1,25 @@
+name: lint
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
```

Comment:
@reiktar Can we keep the python versions we are using to 3.9 so it's consistent across all jobs?

Reply:
This is a effect of despereatley trying to get "act" to work https://github.com/nektos/act But without success. During the testing i was testing many different version of python without any luck.. Changing the python version to 3.9